akvo/rsr/models/report.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+# Akvo RSR is covered by the GNU Affero General Public License.
+# See more details in the license.txt file located at the root folder of the Akvo RSR module.
+# For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ..fields import ValidXMLCharField, ValidXMLTextField
+
+
+class ReportFormat(models.Model):
+    name = ValidXMLCharField(_('name'), max_length=30, unique=True)
+    display_name = ValidXMLCharField(_('display name'), max_length=30, unique=True)
+    icon = ValidXMLCharField(_('icon'), max_length=30)
+
+    def __unicode__(self):
+        return u'{}'.format(self.name)
+
+
+class Report(models.Model):
+    name = ValidXMLCharField(_(u'name'), max_length=100, unique=True)
+    title = ValidXMLCharField(_(u'title'), max_length=200)
+    description = ValidXMLTextField(
+        _(u'description'), blank=True, help_text=_(u'Describe the report.')
+    )
+    url = ValidXMLCharField(
+        _(u'url'),
+        max_length=200,
+        help_text=_(u'Enter the parametrized path for downloading the report'),
+    )
+    parameters = ValidXMLCharField(
```

Comment:
I'm actually going to just remove the `parameters` field, and make it a property so that the `serializer` can return this data, but the user needn't bother about it. `url` field is going to have validation to ensure that the parameters are "valid".